### PR TITLE
yaml

### PR DESCRIPTION
**Pull request by vinugupta**
Added yams

### CODE REVIEW

**Comment by vinugupta** [comment]
cc

**Comment by vinugupta** [comment]
dd

**Comment by tinyPinch** [comment]
better luck next time.

File: vinu.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: vinugupta
+pageUrl: vinugupta.github.io
+timestamp: TODAYS-2017-28-03
+---
```
Comment (hectorsector) [inline vinu.yaml]: @vinugupta you'll need to remove `TODAYS-` and add a line break after line 5. That should allow your tests to pass 🎉

**Comment by crichID** [comment]
@vinugupta it looks like the tests are failing because your web page no longer exists or pages has not been enabled on the repository. If you have created your repository, please go to the repository **settings** and scroll down to the **pages** section. Using the drop-down, select the **master** branch to render your website. When you have finished this, your web page will display at the link shown at the top of the pages section. 

I am going to go ahead and close this pull request since your page is not currently enabled. If you would like to finish this activity, please feel free to re-open this pull request after you have enabled pages on your repository.

**Pull request closed without merging**